### ISSUE TITLE
unexpected EOF error for empty python files

### ISSUE THREAD

**Issue by smacker** [bblfsh/python-driver#184]
Code from https://github.com/bblfsh/bblfshd/issues/264:
/Users/smacker/tmp/srcd-all/apollo/apollo/__init__.py:unexpected EOF
 
not sure if it's a problem in driver or client or sdk.

**Comment by creachadair** [comment]
> For some reason was not able to reproduce it using `bblfsh-cli`.

Hmm. I would have expected the Go client to use the same logic as `bblfshctl`. Does `bblfsh-cli` access a different RPC endpoint than `bblfshctl` uses?

**Comment by bzz** [comment]
> > For some reason was not able to reproduce it using `bblfsh-cli`.
> 
> Hmm. I would have expected the Go client to use the same logic as `bblfshctl`. Does `bblfsh-cli` access a different RPC endpoint than `bblfshctl` uses?

Indeed, it seems like `bblfshctl` [uses UASTv1](https://github.com/bblfsh/bblfshd/blob/master/cmd/bblfshctl/cmd/base.go#L12) protocol, while `bblfsh-cli` uses UASTv2.

**Comment by creachadair** [comment]
> Indeed, it seems like `bblfshctl` [uses UASTv1](https://github.com/bblfsh/bblfshd/blob/master/cmd/bblfshctl/cmd/base.go#L12) protocol, while `bblfsh-cli` uses UASTv2.

Was this intentional, or did we just never get around to bringing them in sync? This could explain several odd differences I've encountered when trying to reproduce bug reports.

**Comment by bzz** [comment]
> > Indeed, it seems like `bblfshctl` [uses UASTv1](https://github.com/bblfsh/bblfshd/blob/master/cmd/bblfshctl/cmd/base.go#L12) protocol, while `bblfsh-cli` uses UASTv2.
> 
> Was this intentional, or did we just never get around to bringing them in sync? This could explain several odd differences I've encountered when trying to reproduce bug reports.

I think the latter one - nobody uses it, so we never got around to syncing them properly.

**Comment by dennwc** [comment]
We still suggest using `bblfshctl` in the documentation, so we should definitely sync it. I'll take care of it.
